src/praisonai/praisonai/cli/commands/gateway.py

```
@@ -130,16 +153,132 @@ def gateway_status(
+    async def _probe_one(name: str, ch_cfg: dict):
+        platform = ch_cfg.get("platform", name)
+        token = _resolve_env_token(ch_cfg.get("token", ""))
+        extras = {
+            k: _resolve_env_token(v)
+            for k, v in ch_cfg.items()
+            if k not in ("platform", "token")
+        }
+        try:
+            bot = Bot(platform, token=token, **extras)
+            return name, await bot.probe()
+        except Exception as e:  # pragma: no cover — defensive
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)